src/app/services/bucket/qarnotBucketService.py

```diff
@@ -0,0 +1,26 @@
+from qarnot import connection, bucket
+import os
+
+from app.services.bucket.baseBucketService import BaseBucketService
+from app import config as cfg
+
+
+class QarnotBucketService(BaseBucketService):
+
+    async def upload_file(self, bucket_name: str, bucket_key: str, file_binary: bin):
+        conn = connection.Connection(client_token=cfg.QARNOT_TOKEN)
+        media_bucket = bucket.Bucket(conn, bucket_name)
```

Comment:
To discuss with the other but I think we will upload the files using the directories.
One could think that one device = one directory

That way if one want to list the files for one device, one could just look at the directory. It would prevent some unauthorized accesses.

Reply:
I agree this might come helpful for devices, but let's explore this in another PR and the directory upload is not especially required for the incoming demo :)